wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -3916,6 +3921,28 @@ func DebugRemoteInstanceCommand(ctx context.Context, pk *scpacket.FeCommandPacke
 	return update, nil
 }
 
+func ClearSudoCache(ctx context.Context, pk *scpacket.FeCommandPacketType) (rtnUpdate scbus.UpdatePacket, rtnErr error) {
+	ids, err := resolveUiIds(ctx, pk, R_Session|R_Screen|R_Remote)
+	if err != nil {
+		return nil, err
+	}
+	ids.Remote.MShell.ClearCachedSudoPw()
+
+	clearAll := resolveBool(pk.Kwargs["all"], false)
+	if clearAll {
+		for _, proc := range remote.GlobalStore.Map {
```

Comment:
you can't iterate directly over GlobalStore.Map (it needs to be synchronized with GlobalStore.Lock).  instead call `remote.GetRemoteMap()` (which makes a copy of the map using the lock for protection).  you can then safely iterate over the copy.

Reply:
fixed